index.html

```diff
@@ -0,0 +1,332 @@
+                <div class="left-block">
+                    <div class="project">
+                        <div class="project-title">                         
+                            <a href="#" class="title">Tonic</a>
+                        </div>
+                        <div class="frame-2">
+                            <a href="#" class="canopy btn-link"> Canopy</a>
+                            <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+                            <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+                            <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                            <a href="#" class="year btn-link"> 2015 </a>
+                        </div>
+                    </div>
+                    <div class="work-detail">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+    
+                    <div class="tags">
+                        <span class="tag">html</span>
+                        <span class="tag">css</span>
+                        <span class="tag">javascript</span>
+                    </div>
+                    <div class="action">
+                        <button type="button" class="btn"><span>See Project</span> </button>
+                    </div>
+                </div>
+            </div>
+
+            <!-----End of card-works for Tonic------->
+
+            <!-----card-works for multi-post stories------->
+            <div class="card-works multipost-stories">
+                <img src="./images/Snapshoot-Portfolio-2.png" alt="Portfolio Snapshot 2" class="card-work multipost-stories">
+
+                <div class="left-block">
+                    <div class="project">
+                        <div class="project-title">                         
+                            <a href="#" class="title">Multi-purpose stories</a>
+                        </div>
+                        <div class="frame-2">
+                            <a href="#" class="canopy btn-link"> Canopy</a>
+                            <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+                            <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+                            <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                            <a href="#" class="year btn-link"> 2015 </a>
+                        </div>
+                    </div>
+                    <div class="work-detail multipost-stories">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+
+                    <div class="tags multipost-stories">
```

Comment:
- [ ] Kindly make sure that all your cards look exactly like the figma template. Consider adding a border to match the Figma template.

- [ ] Kindly align cards to be at the center of the page to match the Figma template

- [ ] Please make sure you style the list of technologies(html, css, javascript) on the card and also the button to match the Figma template.
*Your card*
![3](https://user-images.githubusercontent.com/94551484/186518385-c70441e3-d70c-4684-b00d-fe044ab9754b.png)

*Figma template*
![ccc](https://user-images.githubusercontent.com/94551484/186518425-4f68b404-0c0a-4d93-95b6-ce461f0babac.png)